imports/components/player_list.jsx

```diff
@@ -0,0 +1,44 @@
+export class PlayerList extends React.Component {
+  render() {
+    var players = this.props.players.map(function(player) {
+      return (<PlayerListEntry key={player._id} {...player} />);
+    });
+    var logIn = null;
+    if (!this.props.loggedIn) {
+      logIn = <li>
+        <strong>Log In</strong> to join this list
+      </li>
```

Comment:
I just got it running locally, with a million errors (based off airbnb config). Once I figure out how to prune the rules or add a TODO list I'll push it.